aws_lambda_powertools/utilities/data_classes/dynamo_db_stream_event.py

```diff
@@ -1,169 +1,78 @@
 from enum import Enum
-from typing import Any, Dict, Iterator, List, Optional, Union
+from typing import Any, Dict, Iterator, Optional
 
 from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
 
 
-class AttributeValueType(Enum):
-    Binary = "B"
-    BinarySet = "BS"
-    Boolean = "BOOL"
-    List = "L"
-    Map = "M"
-    Number = "N"
-    NumberSet = "NS"
-    Null = "NULL"
-    String = "S"
-    StringSet = "SS"
-
-
-class AttributeValue(DictWrapper):
-    """Represents the data for an attribute
-
-    Documentation:
-    --------------
-    - https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_streams_AttributeValue.html
-    - https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html
+class TypeDeserializer:
+    """
+    This class deserializes DynamoDB types to Python types.
+    It based on boto3's DynamoDB TypeDeserializer found here:
+    https://boto3.amazonaws.com/v1/documentation/api/latest/_modules/boto3/dynamodb/types.html
+    Except that it deserializes DynamoDB numbers into strings, and does not wrap binary
+    with a Binary class.
```

Comment:
any particular reason why we should not handle Binary (`B`, `BS`) too?

Reply:
We do actually, but we do not wrap it in a `Binary` class like the one boto3 provides here:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/customizations/dynamodb.html#boto3.dynamodb.types.Binary

Maybe the wording can be improved. Any suggestions?